### ISSUE TITLE
declaration-colon-space-after must have not-only-single-space option

### ISSUE THREAD

**Issue by fliptheweb** [stylelint/stylelint#797]
Rule `declaration-colon-space-after` must have not-only-single-space option, e.g. for Sass variables indention:
```sass
$color-font:           #4A4A4A;
$color-secondary-text: #919297;
$color-table-tr:       #F3F6F9;
$color-table-border:   #DDE1E4;
$color-blockquote:     #909297;
```
Or new rules for Sass variables.

Because `stylelint 4.4.0` throw warning with `declaration-colon-space-after: 'always'`:
```sh
Expected single space after ":" (declaration-colon-space-after)
```

PS. if I didn't miss that rules in docs again 😅

**Comment by davidtheclark** [comment]
Duplicate of https://github.com/stylelint/stylelint/issues/769

**Comment by jeddy3** [comment]
There is an [accepted proposal](https://github.com/stylelint/stylelint/issues/769) for this over in #769. I think it'd make a great first rule to try if you'd like to contribute :)

> e.g. for Sass variables indention

If you're wanting to *only* align SCSS variables and do something different for standard declarations, then that's the perfect situation for [building your own plugin](https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/plugins.md) and making use of [`stylelint.rules`](https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/plugins.md#stylelintrules). `stylelint.rules` was added with this *exact* scenario in mind i.e. conditionally applying rules.

I think @kristerkari's ['stylelint-scss'](https://github.com/kristerkari/stylelint-scss) repo would make the perfect home for the such a plugin. Have a chat with him over there if you'd like to contribute.
